### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Yodaazul**
Comentario PR Sergey 1

### CODE REVIEW

**Review by derpy-zombie — APPROVED** [review]
grammar

File: README.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 testing integrations
 for kanbanaize
 to update the current integration.
-random text - no esta bien
+random text - no está bien....
```
Comment (erlantzc) [inline README.md:6]:
```suggestion
random text - no está bien.... eso de no comer pizza
```

**Review by derpy-zombie — APPROVED** [review]
importat things

**Comment by erlantzc** [comment]
Always nice to add pizza to branches

**Pull request closed without merging**